user-interface/src/main/java/life/qbic/datamanager/views/projects/overview/ProjectOverviewMain.java

```diff
@@ -75,6 +82,12 @@ public ProjectOverviewMain(@Autowired ProjectCollectionComponent projectCollecti
         System.identityHashCode(projectCollectionComponent)));
   }
 
+  private boolean isOfferSearchAllowed() {
+    Set<String> allowedRoles = new HashSet<>(Arrays.asList("ROLE_ADMIN", "ROLE_PROJECT_MANAGER"));
+    return SecurityContextHolder.getContext().getAuthentication().getAuthorities().stream()
+        .anyMatch(r -> allowedRoles.contains(r.getAuthority()));
+  }
```

Comment:
NIT: 
Another alternative could be to access and store the user authorities before the user enters the main page via the BeforeEnterEvent and then check these stored events for the right authorities.